### PR TITLE
feat: add legacy `require` API to global

### PR DESCRIPTION
**Pull request by antongolub**
- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

**Comment by antongolub** [comment]
@googlebot I signed it!

**Comment by antonmedv** [comment]
Is there a chance it will be deprecated?

File: zx.mjs [hunk 1]
```diff
@@ -30,6 +31,7 @@ Object.assign(global, {
   chalk,
   fs,
   os,
+  require: createRequire(import.meta.url)
```
Comment (antonmedv) [inline zx.mjs:34]: So require will be relative of zx bin? 

Maybe make it relative to the executed script?

Comment (antongolub) [reply zx.mjs:34]: 1) I _believe_  there was a reason why `createRequire` was added. I have not seen any plans to remove this API.
2) Reasonable.

File: zx.mjs [hunk 2]
```diff
@@ -30,7 +31,7 @@ Object.assign(global, {
   chalk,
   sleep,
   fs,
-  os,
+  os
```
Comment (antonmedv) [inline zx.mjs:34]: Let’s keep tail ,

Comment (antongolub) [reply zx.mjs:34]: My mistake. Reverted.

**Pull request merged**